man/systemd-keyutil.xml

```diff
@@ -71,7 +71,8 @@
         <listitem><para>This command embeds the PKCS#1 signature (RSA) provided with
         <option>--signature=</option> in a PKCS#7 signature using the certificate given with
         <option>--certificate=</option> and writes it to the file specified with <option>--output=</option>
-        in PKCS#7 format (p7s).</para>
+        in PKCS#7 format (p7s). If <option>--content</option> is provided it is included in the p7s,
```

Comment:
in case the content is provided, the correct extension is `p7b`, so this should say something like "it is included in the output (p7b)"

Reply:
> in case the content is provided, the correct extension is p7b

I don't think that is correct, e.g. https://en.wikipedia.org/wiki/PKCS_7#File_types

I believe a `p7b` is a `bundle` that contains multiple certificates, but not signature(s) for data. I think a `p7s` contains signature(s) and, optionally, the corresponding data.